InstallationGuide.md

```diff
@@ -1,50 +1,39 @@
 # Installation Guide
-## Product DFT
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+For more details, please refer configuration section from [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the app version/chart version is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+ 
+ As part of argo CD deployment using helm chart the postgres database dependency will get provide automatic but for EDC, DigitalTwin and Portal you need to provide valid details as per configuration requirement other wise SDE service will get started with default configuration but will not work as expected.
+
+### RUN SDE Backend in k8s cluster
+#### Prerequisites
+- k8s cluster/ minikube
+- helm
+- Docker
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+
+ helm repo add sde-backend https://github.com/eclipse-tractusx/dft-backend/tree/main/charts
+   
+ helm install release-name eclipse-tractusx/dft-backend
+
+### RUN SDE Backend Locally
+#### Prerequisites
+- JDK18
+- Postgres 11.9.13
+
+#### Steps
+1. Clone the GitHub Repository - https://github.com/eclipse-tractusx/dft-backend.
+2. Get your instance of postgres running (Create **dftdb** new database).
+3. Setup your project environment to JDK 18.
+4. Provide require application configuration in application.properties as specified in step configuration.properties.
```

Comment:
If you just use 1., 1. 1. markdown will auto numerate it and you don't need to manually reenumerate this when updating the list.

You should also try to avoid . at the end of a list senetence

Reply:
We have check auto numerate is working .